hawc/apps/animalv2/models.py

```diff
@@ -0,0 +1,576 @@
+from django.core.validators import MinValueValidator
+from django.db import models
+
+from ..assessment.models import BaseEndpoint, DSSTox
+from ..vocab.models import Term
+from . import constants, managers
+
+"""
+Generate a GraphViz dot file of this model:
```

Comment:
Move this content to the appropriate location in `docs/docx/database.md`; we can all the static image later after things have stabilized